scheduledwf-core/src/main/java/io/github/jas34/scheduledwf/exception/ApplicationException.java

```diff
@@ -0,0 +1,93 @@
+package io.github.jas34.scheduledwf.exception;
+
+/* copied from
+ * Copyright 2020 Netflix, Inc.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+@SuppressWarnings("serial")
+public class ApplicationException extends RuntimeException {
```

Comment:
It will make sense to follow the implementation pattern of exception handling done by conductor. Let's solve this as a separate ticket. I will open the same.

Reply:
Please create a separate ticket.